src/partials/partial.erb.html

```diff
@@ -1,4 +1,4 @@
-<div data-key="lmn-book" data-name="<%= customisation.name %>"
+<div data-icons="true" data-key="lmn-book" data-name="<%= customisation.name %>"
```

Comment:
Why have you added this to the div? What's wrong with it being in the options like everything else?